templates/netlify/package.json

```diff
@@ -10,7 +10,7 @@
     "start": "cross-env NODE_ENV=production netlify dev"
   },
   "dependencies": {
-    "@netlify/functions": "^0.10.0",
+    "@netlify/functions": "latest",
     "@remix-run/netlify": "*",
     "@remix-run/node": "*",
```

Comment:
We should probably replace this with `@remix-run/server-runtime` when choosing for `"Netlify Edge Functions"` in `remix.init`